pyproject.toml

```diff
@@ -22,6 +22,9 @@ dependencies = [
     "scanspec",
     "PyYAML",
     "click",
+    "fastapi",
+    "uvicorn",
+    "httpx",
```

Comment:
Should this be `requests` instead of `httpx`? It doesn't look like anything uses httpx but requests is used

Reply:
Here's what happens if you do that, looking at CI test jobs:

```
ImportError while loading conftest '/home/runner/work/blueapi/blueapi/tests/conftest.py'.
tests/conftest.py:4: in <module>
    from fastapi.testclient import TestClient
/opt/hostedtoolcache/Python/3.9.[16](https://github.com/DiamondLightSource/blueapi/actions/runs/4927983607/jobs/8805745964?pr=135#step:6:17)/x64/lib/python3.9/site-packages/fastapi/testclient.py:1: in <module>
    from starlette.testclient import TestClient as TestClient  # noqa
/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/starlette/testclient.py:16: in <module>
    import httpx
E   ModuleNotFoundError: No module named 'httpx'
```

Reply:
have now fixed this; turns out I just had to change `fastapi` to `fastapi[all]` in the pyproject.toml.